### ISSUE TITLE
Add [auto-dark]

### ISSUE THREAD

**Issue by jrappen** [wbond/package_control_channel#7846]
## https://github.com/jrappen/sublime-auto-dark

> Please see repo for code.

---

I'll add the following code to my channel file once approved at:

https://github.com/jrappen/sublime-packages

```json
        {
            "name": "auto-dark",
            "author": "Johannes Rappen (jrappen)",
            "details": "https://github.com/jrappen/sublime-auto-dark",
            "donate": "https://paypal.me/jrappen",
            "labels":
            [
                "dark-mode",
                "automation"
            ],
            "releases":
            [
                {
                    "sublime_text": ">=4065",
                    "tags": true
                }
            ]
        }
```

**Comment by FichteFoll** [comment]
I personally strongly dislike with your abuse of a class for a namespace, since that's what modules are for. Don't use classes when you don't need to track state in its instances. In fact, you don't even need a separate module here since there isn't anything else in `src.auto_dark` anyway.

Per Python conventions, ALL_CAPS style should be used for constants only (I can see an argument for the package name), not for read-only variables.

It doesn't look like you support Linux atm, so you should not allow installation on for that platform.

LGTM otherwise.

**Comment by jrappen** [comment]
closed in jrappen/sublime-packages@a277361

**Comment by jrappen** [comment]
@FichteFoll thanks, appreciate the feedback
